packaging/PLATFORM_SUPPORT.md

```diff
@@ -0,0 +1,153 @@
+  effort to ensure that things keep working on them either. Additionally, Netdata may occasionally move an
+  older supported platform to this tier if we can no longer provide full support for it for technical reasons.
+  Users reporting bugs on previously supported platforms will be asked to reproduce the issue on a currently
+  supported platform, and if they cannot the issue will be closed.
+
+Any platforms not listed in any of these categories may or may not work
+
+> ### What happens when a platform becomes ‘Previously Supported’ for technical reasons?
+>
+> In general, if we determine that we need to drop support for a given platform due to technical limitations,
+> this will be announced in the release notes of the next stable release with a deprecation notice. The platform
+> will be supported for _that release_, and will be removed from nightlies some time before the next release.
+
+> ### What constitutes ‘End of Life’ for a platform?
+>
+> We consider a platform to be end-of-life when the upstream maintainers of that platfrom stop providing official
+> support for it themselves, or when that platform transitions into an ‘extended security maintenance’ period.
+>
+> Platforms that meet these criteria will be immediately transitioned to the **Previously Supported** category,
+> with no prior warning from Netdata and no deprecation notice, unlike those being dropped for technical reasons,
+> as our end of support should already coincide with the end of the normal support lifecycle for that platform..
+
+A general outline of the various support tiers and categories is shown in the following table
+
+|   | Bug Support| Guaranteed Configurations | CI Coverage | Native Packages | Static Build Support |
+| - | -------------------- | ------------------------- | ----------- | -------------- | -------------------- |
+| Core | High priority | Everything but rare edge cases | Full | Yes, if we can provide them | Full |
+| Intermediate | Normal priority | Common cases | Partial (CI mostly equivalent to **Core**, but possibly with some gaps, and not required to pass) | Possibly | Full |
+| Community | Best Effort | Default only | None | No | Best Effort |
+| Third-party Supported | Users directed to platform maintainers | None | None | No | Best Effort |
+| Previously Supported | Users asked to upgrade | None | None | Yes, but only already published versions | Best Effort |
+
+- ‘Bug Support’: How we handle of platform-specific bugs.
+- ‘Guaranteed Configurations’: Which runtime configurations for the agent we try to guarantee will work with minimal effort from users.
+- ‘CI Coverage’: What level of coverage we provide for the platform in CI.
+- ‘Native Packages’: Whether we provide native packages for the system package manager for the platform.
+- ‘Static Build Support’: How well our static builds are expected to work on the platform.
+
+# Currently supported platforms
```

Comment:
How about a list of currently supported architectures before that section?

Reply:
Barring some occasional 32/64-bit cleanness issues, we don’t do anything that is architecture dependent in our code (I’ve double checked that we build properly on a bunch of architectures, including exotic stuff like PA-RISC and SuperH), so such a list would only matter for pre-built packages. We clearly document per-platform what, if any, architectures we provide native packages for, and we also clearly document in in the description of static builds in the overhauled kickstart document what platforms those are built for (though that bit actually needs updated), so I see little to no value in providing a separate list of supported CPU architectures.